tool-populate/api/helpers.js

```diff
@@ -0,0 +1,21 @@
+var Parse = require('parse/node')
+/*
+  Accepts Parse object and sets ACL giving users in Admin Role read/write access
+  @param [Object] obj, Parse object on which to modify ACL
+ */
+function setAdminACL(obj) {
+  var query = new Parse.Query(Parse.Role)
+  query.equalTo('name', 'admin')
+  return query.first({useMasterKey: true})
+    .then(function(adminRole) {
+      var acl = new Parse.ACL()
```

Comment:
I forgot to run `-i` to create the initial roles and the script had a bunch of errors due to adminRole being undefined.  Up to you if want to handle this or not.